src/ChannelServer/World/SpawnZone.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Melia.Shared.Data.Database;
+using Melia.Shared.Util;
+using Melia.Shared.World;
+
+namespace Melia.Channel.World
+{
+
+
+	public struct SpawnData
+	{
+		public string spawnName;
+		public string mapName;
+		public string monsterName;
+		public int count;
+		public int countVariation;
+		public bool isFixedLocation;
+		public Position Position { get; set; }
+		public Direction Direction { get; set; }
+		public int respawnTime;
+		public int xMin, xMax, zMin, zMax;
+	}
+
+	public class SpawnZone
+	{
+
+		public int Id;
+		public SpawnData Data { get; set; }
```

Comment:
Those two properties should probably have private setters or no setter at all, so they can't be accidentally changed from outside.